tests/universe/InstallPackageModal-cy.js

```diff
@@ -14,30 +14,8 @@ describe('Install Package Modal', function () {
 
   it('displays install modal for package', function () {
     cy
-      .get('.modal .form-element-inline-text')
-      .should('contain', 'marathon-user');
-  });
-
-  it('displays config header for package', function () {
```

Comment:
There's time allocated for people (you) to write more tests for Universe. This can be done then.